### PR TITLE
1212

### PR DESCRIPTION
**Pull request by ARMOKS**
## Requirements

- [ ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ ] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/_Goobstation/Entities/Clothing/OuterClothing/armour.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@
   - type: Armor
     modifiers:
       coefficients:
-        Blunt: 0.60
+        Blunt: 0.61
```
Comment (ARMOKS) [inline Resources/Prototypes/_Goobstation/Entities/Clothing/OuterClothing/armour.yml:85]: yeah

**Comment by HenhenIII** [comment]
This is peak

**Comment by JoeHammad1844** [comment]
This is peak

**Comment by whateverusername0** [comment]
This is peak

**Comment by Piras314** [comment]
This is peak

**Comment by Piras314** [comment]
kaep si sihT

**Pull request closed without merging**